src/main/java/org/example/MainServlet.java

```diff
@@ -1,19 +1,50 @@
 package org.example;
 
-import jakarta.servlet.annotation.WebServlet;
 import jakarta.servlet.http.HttpServlet;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
+import org.example.model.UrlPattern;
+import org.example.service.*;
+
 import java.io.IOException;
-import java.io.PrintWriter;
 
 public class MainServlet extends HttpServlet {
 
+    private final HelloService helloService = HelloService.getInstance();
+    private final TableService tableService = TableService.getInstance();
+    private final NotFoundService notFoundService = NotFoundService.getInstance();
+    private final JsonObjectService jsonObjectService = JsonObjectService.getInstance();
+
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
```

Comment:
Check response codes in the HTTP.
What response code should we send on success and on not found?

Reply:
fixed